docs/v0.21/testing-0.21rc2.md

```diff
@@ -96,33 +97,32 @@ $ tor
 ```
 
 ### 2. Manual Testing
-For those wanting to dig deeper, [Bitcoin Core provides documentation](https://github.com/bitcoin/bitcoin/blob/master/doc/tor.md) on how to test running a node on Tor. There is little manual config to be done. In fact, on some linux distros if there is a tor daemon running on the machine bitcoind will pick it up and authenticate with a cookie file.
+For those wanting to dig deeper, [Bitcoin Core provides documentation](https://github.com/bitcoin/bitcoin/blob/master/doc/tor.md) on how to test running a node on Tor. There is little manual config to be done. In fact, on some Linux distros, if there is a Tor daemon running on the machine bitcoind will pick it up and authenticate with a cookie file.
 
 #### Manual Testing with GUI
 
 #### Listen on torv3
-We are going to setup our node to [automatically listen on Tor](https://github.com/bitcoin/bitcoin/blob/master/doc/tor.md#3-automatically-listen-on-tor). This means that the node is going to look for other peers on the tor network.
+We are going to set up our node to [listen on Tor automatically](https://github.com/bitcoin/bitcoin/blob/master/doc/tor.md#3-automatically-listen-on-tor). This means that the node is going to look for other peers on the Tor network.
 
 ---
 
 ## Testing Signet
-The Bitcoin [testnet](https://en.bitcoin.it/wiki/Testnet) is a [proof-of-work](https://en.bitcoin.it/wiki/Proof_of_work) based testing framework where volunteers are relied on to [mine](https://en.bitcoin.it/wiki/Mining) [blocks](https://en.bitcoin.it/wiki/Block) with real CPU power and in turn receive worthless testnet coins. Since the economics of the the mainnet are not at play here; we get a network that is unpredictable and, frustratingly, unreliable.
+The Bitcoin [testnet](https://en.bitcoin.it/wiki/Testnet) is a [proof-of-work](https://en.bitcoin.it/wiki/Proof_of_work) based testing framework where volunteers are relied on to [mine](https://en.bitcoin.it/wiki/Mining) [blocks](https://en.bitcoin.it/wiki/Block) with real CPU power and in turn receive worthless testnet coins. Since the economics of mainnet are not at play here, we get a network that is unpredictable and, frustratingly, unreliable.
```

Comment:
I think the links here are overkill. You don't need to teach testers what mining or blocks are if they are in the trenches testing a release candidate.

Reply:
removed links in 272bb8d46bfc4ad2b05e12f735f1a4d0fbfdd903